util.mli

```diff
@@ -68,13 +68,15 @@ exception Undefined of string * json
       return undefined. Currently this only happens when an array index is out
       of bounds. *)
 
-val ( |> ) : 'a -> ('a -> 'b) -> 'b
-  (** Forward pipe operator; useful for composing JSON access functions
-      without too many parentheses *)
```

Comment:
@mjambon Ah, I didn't think of that. I'm not sure how you meant by marking it obsolete, like by putting it in the doc string or I've seen some people do the ppx thing, something like `[@@ocaml.deprecated "blah blah"]`.